deps/wazuh_testing/wazuh_testing/modules/aws/data_generator.py

```diff
@@ -94,22 +103,204 @@ def get_data_sample(self) -> dict:
+
+        return f'{path}{name}'
+
+    def get_data_sample(self) -> str:
+        """Returns a sample of data according to the KMS format.
+
+        Returns:
+            str: Syntetic data.
+        """
+        return json.dumps(
+            {
+                'version': '0',
+                'id': str(uuid4()),
+                'detail-type': 'AWS API Call via CloudTrail',
+                'source': 'aws.kms',
+                'account': cons.RANDOM_ACCOUNT_ID,
+                'time': '2018-11-07T17:27:01Z',
+                'region': cons.US_EAST_1_REGION,
+                'resources': [],
+                'detail': {
+                    'eventVersion': '1.05',
+                    'userIdentity': {
+                        'type': 'IAMUser',
+                        'principalId': get_random_string(20),
+                        'arn': f"arn:aws:iam::{cons.RANDOM_ACCOUNT_ID}:user/fake.user",
+                        'accountId': cons.RANDOM_ACCOUNT_ID,
+                        'accessKeyId': get_random_string(20),
+                        'userName': 'fake.user',
+                        'sessionContext': {
+                            'attributes': {
+                                'mfaAuthenticated': 'false',
+                                'creationDate': '2018-11-07T07:53:47Z'
+                            }
+                        },
+                        'invokedBy': 'secretsmanager.amazonaws.com'
+                    },
+                    'eventTime': '2018-11-07T17:27:01Z',
+                    'eventSource': 'kms.amazonaws.com',
+                    'eventName': 'GenerateDataKey',
+                    'awsRegion': cons.RANDOM_ACCOUNT_ID,
+                    'sourceIPAddress': 'secretsmanager.amazonaws.com',
+                    'userAgent': 'secretsmanager.amazonaws.com',
+                    'requestParameters': {
+                        'keySpec': 'AES_256',
+                        'encryptionContext': {
+                            'SecretARN': f"arn:aws:secretsmanager:us-east-1:{cons.RANDOM_ACCOUNT_ID}:secret:test-aws",
+                            'SecretVersionId': str(uuid4())
+                        },
+                        'keyId': 'alias/aws/secretsmanager'
+                    },
+                    'responseElements': None,
+                    'requestID': str(uuid4()),
+                    'eventID': str(uuid4()),
+                    'readOnly': True,
+                    'resources': [
+                        {
+                            'ARN': f"arn:aws:kms:us-east-1:{cons.RANDOM_ACCOUNT_ID}:key/{str(uuid4())}",
+                            'accountId': cons.RANDOM_ACCOUNT_ID,
+                            'type': 'AWS::KMS::Key'
+                        }
+                    ],
+                    'eventType': 'AwsApiCall',
+                    'vpcEndpointId': f"vpce-{get_random_string(17)}"
+                }
+            }
+        )
```

Comment:
For the reviewer, please disregard this already has been fixed in https://github.com/wazuh/wazuh-qa/pull/3715/.